lib/mem.h

```diff
@@ -63,7 +63,17 @@ extern "C" {
 /*-**************************************************************
 *  Basic Types
 *****************************************************************/
-#if defined (__cplusplus) || (defined (__STDC_VERSION__) && (__STDC_VERSION__ >= 199901L) /* C99 */)
+#if defined(_MSC_VER) && (_MSC_VER < 1400)     /* For VC6 and other legacy versions */
+  typedef unsigned char       BYTE;
+  typedef unsigned short      U16;
+  typedef   signed short      S16;
+  typedef unsigned int        U32;
+  typedef   signed int        S32;
+  typedef unsigned __int64    U64;
+  typedef   signed __int64    S64;
+  #define LIT_S64(x) x##i64
```

Comment:
Makes sense and will take care of this. Although "literals" is the precise language term, in context of ZSDT it's really kind of confusing.